### PR TITLE
Log path and bcaf

### PR DESCRIPTION
**Pull request by carlhejiayu**
make sure log path clear and consistent everywhere

### CODE REVIEW

File: installing.html.md.erb [hunk 1]
```diff
@@ -526,7 +526,7 @@ To configure <%= vars.product_abbr %> to monitor containers on the Kubernetes wo
 ##<a id="alerts"></a> Configure Forwarding for <%= vars.product_abbr %> Alerts
 
 <%= vars.product_abbr %> writes all alerts to the BOSH logs for the VMs in your deployment.
-These logs are located at `/var/vcap/sys/log/fim/fim.stdout.log` and
+These logs are located at `/var/log/syslog` in linux system , while in windows these logs located in
```
Comment (DennyHoang) [inline installing.html.md.erb:529]: there is a space before comma

File: log-examples.html.md.erb [hunk 2]
```diff
@@ -14,7 +14,8 @@ system, to verify regular activity and generate alerts for file system operation
 ##<a id="logs"></a> Log Output Destination
 
 <%= vars.product_abbr %> produces many different logs depending on what operation is being performed.
-You can find these logs at `/var/vcap/sys/log/fim/fim.stdout.log`.
+You can find these logs at `/var/log/syslog` in linux system , while in windows these logs located in
```
Comment (DennyHoang) [inline log-examples.html.md.erb:17]: There is a space before the comma

Comment (DennyHoang) [reply log-examples.html.md.erb:17]: "These logs **are** located in"

File: installing.html.md.erb (hunk 1 above)
Comment (DennyHoang) [reply installing.html.md.erb:529]: "These logs **are** located in"

Comment (carlhejiayu) [reply installing.html.md.erb]: fixed

File: log-examples.html.md.erb (hunk 2 above)
Comment (carlhejiayu) [reply log-examples.html.md.erb]: fixed

**Comment by radaeze** [comment]
@DennyHoang @carlhejiayu Thanks!

**Pull request merged**